build/redeemer/Redeemer.sol

```diff
@@ -189,6 +189,30 @@ contract Redeemer {
     return true;
   }
 
+
+  /// @notice implements the redeem method for the contract to fulfil the 
+  /// ERC-5095 interface
+  /// @param u address of the underlying asset
+  /// @param m maturity of the market
+  /// @param f address from where the underlying asset will be burned
+  /// @param t address to where the underlying asset will be transferred
+  /// @return bool true if the underlying asset was burned successfully
+  function redeem(address u, uint256 m, address f, address t) public authorized(admin) returns (bool) {
```

Comment:
so, product wise, this is a different thing than the `Redeemer.redeem` exposed as public interface.

should it's name reflect what it's actually doing? I mean, I am always **fine** with being thrifty with words and names -- but this is a different thing altogether than `redeem` right? A FE / Dapp will probably want this differentiated more than just signature. My gut anyway -- i could be wrong. we can discuss.